OpenRA.Mods.Cnc/Traits/Attack/AttackLeap.cs

```diff
@@ -30,17 +30,17 @@ public class AttackLeapInfo : AttackFrontalInfo, Requires<MobileInfo>
 		public override object Create(ActorInitializer init) { return new AttackLeap(init.Self, this); }
 	}
 
-	public class AttackLeap : AttackFrontal
+	public class AttackLeap : AttackBase
 	{
-		readonly AttackLeapInfo info;
+		public new readonly AttackLeapInfo Info;
```

Comment:
I don't think we use this approach (`new` key) anywhere else, but I don't remember if there was a technical reason for that, maybe @pchote can shed some light on this.

Reply:
It is too long ago now, for me to remember what the reason for choosing this approach was at the time.

Reply:
We mainly don't use this approach for consistency with all the existing code that never used this approach. IMO we probably should start using it, and at some point fix the few existing cases that use variable name gymnastics to avoid it.